app/models/doi.rb

```diff
@@ -585,78 +585,80 @@ def self.gql_query_aggregations(facet_count: 10)
     end
   end
 
-  def self.query_aggregations
-    {
-      resource_types: { terms: { field: "resource_type_id_and_name", size: 16, min_doc_count: 1 } },
-      states: { terms: { field: "aasm_state", size: 3, min_doc_count: 1 } },
-      published: {
-        date_histogram: {
-          field: "publication_year",
-          interval: "year",
-          format: "year",
-          order: {
-            _key: "desc",
+  def self.query_aggregations(facet_count: 10)
+    if facet_count.to_i.positive?
+      {
+        resource_types: { terms: { field: "resource_type_id_and_name", size: facet_count, min_doc_count: 1 } },
```

Comment:
With a global facet count, this will now return different number of resource types, previously 16. Do we have anything that depends on this?

Reply:
Good point. We could hardcode the actual number of possible facets for resource type if facet_count > 0.

Reply:
The reason I use an integer for the fact count instead of a boolean is that the initial issue for GraphQL (#701) had a different purpose, increasing the number of facets. In GraphQL you can show only the facet(s) you want giving you more flexibility.

Do you suggest that we don't allow that flexibility with the REST API, just having facets on or off?

Reply:
Ah I see. Ye I think we should reduce the flexibility for the REST API, it makes things simpler.

Reply:
Ok, will do that.